src/main/frontend/utils.js

```diff
@@ -315,12 +315,14 @@ export const nodePath = Object.assign({}, path, {
   join (input, ...paths) {
     let orURI = null
 
-    try {
-      orURI = new URL(input)
-      input = input.replace(orURI.protocol + '//', '')
-        .replace(orURI.protocol, '')
-        .replace(/^\/+/, '/')
-    } catch (_e) {}
+    if (input.startsWith("file://")) {
```

Comment:
This is a temporary fix.
Better refactor as "safe-url-join" and "safe-path-join"
URL always uses percent-encoding.
While paths are always the same as system paths.